web/scenes/Portal/Teams/TeamId/Apps/AppId/Configuration/AppTopBar/SubmitAppModal/index.tsx

```diff
@@ -57,10 +61,16 @@ export const SubmitAppModal = (props: SubmitAppModalProps) => {
 
   const [validateLocalisation, {}] = useValidateLocalisationMutation();
 
-  const { register, handleSubmit } = useForm<SubmitAppFormValues>({
+  const {
+    register,
+    handleSubmit,
+    watch,
+    formState: { errors },
+  } = useForm<SubmitAppFormValues>({
     resolver: yupResolver(schema),
     defaultValues: {
       is_developer_allow_listing: isDeveloperAllowListing,
+      changelog: "",
```

Comment:
And sanitization

Reply:
ofc, in `web/scenes/Portal/Teams/TeamId/Apps/AppId/Configuration/AppTopBar/server/submit.ts`, because of schema defined in `web/scenes/Portal/Teams/TeamId/Apps/AppId/Configuration/AppTopBar/form-schema.ts`